app/controllers/main/routes.py

```diff
@@ -116,7 +116,29 @@ def RemoveRSVP():
     currentRsvpParticipant.delete_instance()
 
     flash("Successfully unregistered for event!", "success")
-    return redirect(url_for("admin.editEvent", eventId=event.id))
+    return redirect(url_for("admin.editEvent", eventId=eventId, program=program))
```

Comment:
This is wrong. There is no need to pass the program into editEvent. If it's not something you changed, it's likely a merge conflict issue.

Reply:
Notice that it's also using a variable `eventId` which doesn't exist in the surrounding code